pxr/imaging/hdSt/materialNetwork.h

```diff
@@ -20,9 +20,57 @@
 
 PXR_NAMESPACE_OPEN_SCOPE
 
+class SdrRegistry;
 class HdStResourceRegistry;
 using HioGlslfxSharedPtr = std::shared_ptr<class HioGlslfx>;
 using HdSt_MaterialParamVector = std::vector<class HdSt_MaterialParam>;
+struct HdSt_MaterialFilterTask;
+using HdSt_MaterialFilterTaskSharedPtr =
+    std::shared_ptr<HdSt_MaterialFilterTask>;
+
+extern HdMaterialNode2 const*
```

Comment:
Don't think this needs an extern, but if you want it to be publicly visible (which maybe we don't care?) you could throw an HDST_API on it...